src/Security/Confirmation/Storage.php

```diff
@@ -236,7 +238,8 @@ public function getItem($key)
      */
     public function setSuccessRequest(HTTPRequest $request)
     {
-        $this->setSuccessUrl($request->getURL(true));
+        $url = Controller::join_links(Director::baseURL(), $request->getURL(true));
```

Comment:
Similar to above, I think you could probably use `Director::absoluteURL($request->getURL(true))` here

Reply:
Maybe we need $request->getAbsoluteURL()?